scripts/rootpy-add-include-path

```diff
@@ -0,0 +1,23 @@
+#! /bin/env python
```

Comment:
The idea is just to keep the number of installed scripts to an absolute minimum to avoid "clutter". You can basically drop your code into scripts/rootpy after line 424:

https://github.com/rootpy/rootpy/blob/master/scripts/rootpy#L424

with the ``subparsers.add_parser`` modification and the rest of your existing script in a function called ``add_include_path``.